scaffold/Taskfile.yml

```diff
@@ -4,3 +4,4 @@ dotenv: ['.env']
 
 includes:
   drupal: ./vendor/lullabot/drainpipe/tasks/drupal.yml
+  snapshot: ./vendor/lullabot/drainpipe/tasks/snapshot.yml
```

Comment:
A next-day thought; is "artifact" or "archive" a more common term than "snapshot"? Especially as we use snapshot in the context of "development snapshot".